Mathlib/NumberTheory/ModularForms/QExpansion.lean

```diff
@@ -0,0 +1,128 @@
+/-
+Copyright (c) 2024 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+import Mathlib.Analysis.Complex.Periodic
+import Mathlib.NumberTheory.ModularForms.Basic
+import Mathlib.NumberTheory.ModularForms.Identities
+
+/-!
+# Q-expansions of modular forms
```

Comment:
```suggestion
# q-expansions of modular forms
```
This is just a random suggestion, if you have any knowledge of a typographic standard that says `q` should be capitalized here just ignore this comment.